docs/conflicts.rst

```diff
@@ -0,0 +1,131 @@
+
+Note that unlike tools like Git, we do not examine the contents of the file or try to produce differences -- everything is determined from the metadata in the Snapshot.
+This means that even if we happened to make the very same edits "at the same time" it would still be a conflict.
+
+
+Showing Conflicts Locally
+-------------------------
+
+Once a conflict is detected, "conflict marker" files are put into the local magic folder location (our local file remains unmodified, and something like ``foo.conflict-desktop`` will appear.
+The state database is also updated (conflicts can also be listed via the API and CLI).
+
+Although magic-folder itself doesn't try to examine the contents, you can now use any ``diff`` or similar tools you prefer to look at what is different between your copy and other participant(s) copies.
+
+
+Resolving Conflicts
+-------------------
+
+We cannot "magically" solve a conflict: two devices produced new Snapshots while not communicating with each other.
+
+Thus, it is up to the humans using these devices to determine what happens.
+Once appropriate changes are decided upon, a new Snapshot is produced with *two or more parents*: one parent for each of the Snapshots involved in the conflict (we've only talked about one other participant so far, but there could be more).
+
+Such a Snapshot (with two or more parents) indicates to the other clients a particular resoltion to the conflict has been decided.
+
+So there's actually another case when we see an incoming new Snapshot: it may in fact *resolve an existing* conflict.
+If this is the case, conflict markers are removed and the local database is updated (i.e. removing the conflict).
+
+It is a human problem if this resolution is not to your particular liking; you can produce an edit again or talk to the human who runs the other computer(s) involved.
+The history of these changes *is available* in the parent (or grandparent) Snapshots if the UX you're using can view or restore these.
+
+
+Resolving via Filesystem
+------------------------
+
+Not currently possible.
+
+
+Resolving via the CLI
+---------------------
+
+The subcommand ``magic-folder resolve`` may be used to specify a resolution.
+It allows you to choose ``--mine`` or ``--theirs`` (if there is only one other conflict).
+Otherwise, you must apply the ``--use <name>`` option to specify which version to keep.
+
+Currently there is no API for doing something more complex (e.g. simultaneuously replacing the latest version with new content).
+
+Complete example:
+
+.. code-block:: console
+
+    magic-folder resolve --mine ~/Documents/Magic/foo
+
+
+Resolving via the HTTP API
+--------------------------
+
+See :ref:`api_resolve_conflict`
+
+
+Future Directions
+-----------------
+
+We do not consider the current conflict functionality "done".
+There are other features required to make this more robust and have a nicer user experience.
```

Comment:
Minor: maybe that terminating period (".") should be a colon (":") to signify that you're hereby listing those other features.